examples/ginkgo-overhead/doc/results.dox

```diff
@@ -0,0 +1,15 @@
+<h1>Results</h1>
+This is the expected output:
+
+@code{.cpp}
+This is Ginkgo 0.0.0 (develop)
+    running with core module 0.0.0 (develop)
+    the reference module is  0.0.0 (develop)
+    the OpenMP    module is  0.0.0 (develop)
+    the CUDA      module is  not compiled
+Running 1000000 iterations of the CG solver took a total of 1.50535 seconds.
+        Average library overhead:     1505.35 [nanoseconds / iteration]
+@endcode
```

Comment:
@tcojean I meant that we should change this part (for the other examples as well) at some point before the release (If it is not automatically changed, but I don't think that that is necessary).